WalletWasabi.Tests/UnitTests/MempoolMirrorTests/SpentLabelTests.cs

```diff
@@ -0,0 +1,70 @@
+using NBitcoin;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.BitcoinCore.Rpc;
+using WalletWasabi.Tests.XunitConfiguration;
+using WalletWasabi.Tor.Http;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.MempoolMirrorTests
+{
+	[Collection("RegTest collection")]
+	public class SpentLabelTests
+	{
+		public SpentLabelTests(RegTestFixture regTestFixture)
+		{
+			RegTestFixture = regTestFixture;
+			BackendHttpClient = regTestFixture.BackendHttpClient;
+			BackendApiHttpClient = new ClearnetHttpClient(regTestFixture.HttpClient, () => RegTestFixture.BackendEndPointApiUri);
+		}
+
+		public RegTestFixture RegTestFixture { get; }
+		public IHttpClient BackendHttpClient { get; }
+		private IHttpClient BackendApiHttpClient { get; }
+
+		[Fact]
+		public async Task CanGetBackSpenderTransactionAsync()
+		{
+			(_, IRPCClient rpc, _, _, _, _, _) = await RegressionTests.Common.InitializeTestEnvironmentAsync(RegTestFixture, 0);
+
+			var network = rpc.Network;
+
+			using var k1 = new Key();
+			var blockId = await rpc.GenerateToAddressAsync(1, k1.PubKey.WitHash.GetAddress(network));
+			var block = await rpc.GetBlockAsync(blockId[0]);
+			var coinBaseTx = block.Transactions[0];
+
+			var tx = Transaction.Create(network);
+			using var k2 = new Key();
+			tx.Inputs.Add(coinBaseTx, 0);
+			tx.Outputs.Add(Money.Coins(49.9999m), k2.PubKey.WitHash.GetAddress(network));
+			tx.Sign(k1.GetBitcoinSecret(network), coinBaseTx.Outputs.AsCoins().First());
+			var valid = tx.Check();
+
+			await rpc.GenerateAsync(101);
+
+			using StringContent content = new($"'{tx.ToHex()}'", Encoding.UTF8, "application/json");
+			using var response = await BackendApiHttpClient.SendAsync(HttpMethod.Post, "btc/blockchain/broadcast", content);
```

Comment:
`response` isn't used anywhere else, you don't have to declare a variable for it.

Reply:
Done